.vscode/launch.json

```diff
@@ -11,12 +11,31 @@
 				"--extensionDevelopmentPath=${workspaceFolder}"
 			],
 			"env": {
+				"AZCODE_DOCKER_IGNORE_BUNDLE": "1",
 				"DEBUGTELEMETRY": "1"
 			},
 			"stopOnEntry": false,
 			"sourceMaps": true,
 			"outFiles": [
-				"${workspaceFolder}/../dist/**/*.js"
+				"${workspaceFolder}/**/*.js"
```

Comment:
Do breakpoint work?  I had trouble with them until I set this to ${workspaceFolder}/out/**/*.js

Reply:
Works for me, but I can add that